tests/foreman/ui_airgun/test_contentcredential.py

```diff
@@ -361,6 +361,51 @@ def test_positive_add_product_using_repo_discovery(session, gpg_path):
                 == repo_name)
 
 
+@tier2
+def test_positive_add_product_and_search(session, module_org, gpg_content):
+    """Create gpg key with valid name and valid gpg key
+    then associate it with custom product that has one repository
+    After search and select product through gpg key interface
+
+    :id: 0bef0c1b-4811-489e-89e9-609d57fc45ee
+
+    :customerscenario: true
+
+    :expectedresults: Associated product can be found and selected through
+        gpg key 'Product' tab
+
+    :BZ: 1411800
+
+    :CaseLevel: Integration
+    """
+    name = gen_string('alpha')
+    gpg_key = entities.GPGKey(
+        content=gpg_content,
+        name=name,
+        organization=module_org,
+    ).create()
+    # Creates new product and associate GPGKey with it
+    product = entities.Product(
+        gpg_key=gpg_key,
+        organization=module_org,
+    ).create()
+    # Creates new repository without GPGKey
+    entities.Repository(
+        url=FAKE_1_YUM_REPO,
+        product=product,
+    ).create()
+    with session:
+        values = session.contentcredential.read(gpg_key.name)
+        assert len(values['products']['table']) == 1
+        assert values['products']['table'][0]['Name'] == product.name
```

Comment:
just a thought , can you please add the `assertion` to see the `Repositories` tab details as well ?